### PR TITLE
feat: moved invite members form to antd form

### PR DESCRIPTION
**Pull request by GermaVinsmoke**
Solves feature issue - invite member form of #2150 
Close - #2659

### CODE REVIEW

File: frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx [hunk 1]
```diff
@@ -1,107 +1,81 @@
...
+								<Space key={key} direction="horizontal">
+									<Form.Item
+										name={[name, 'email']}
+										rules={[{ required: true, message: 'Missing email' }]}
+									>
+										<Input placeholder={t('email_placeholder')} />
+									</Form.Item>
+									<Form.Item
+										name={[name, 'name']}
+										rules={[{ required: true, message: 'Missing name' }]}
```
Comment (palashgdev) [inline frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx:46]: name is optional here

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx [hunk 2]
```diff
@@ -1,107 +1,81 @@
...
+											<Option value="ADMIN">ADMIN</Option>
+											<Option value="VIEWER">VIEWER</Option>
+											<Option value="EDITOR">EDITOR</Option>
+										</SelectDrawer>
+									</Form.Item>
+								</Space>
+							))}
+							<Form.Item>
+								<Button
+									onClick={(): void => add()}
```
Comment (palashgdev) [inline frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx:61]:
```suggestion
									onClick={add}
```

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx (hunk 2 above)
Comment (GermaVinsmoke) [reply frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx:61]: Isn't this gonna complain about that eslint error - no function return type?

Comment (palashgdev) [reply frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx:61]: nopes

Comment (GermaVinsmoke) [reply frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx]: Okay, resolved

File: frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx (hunk 1 above)
Comment (GermaVinsmoke) [reply frontend/src/container/OrganizationSettings/InviteTeamMembers/index.tsx]: Made this field optional

**Review by palashgdev — APPROVED** [review]

**Pull request merged**